### DISCUSSION TITLE
AWS Amplify + Prisma ("ENOENT: no such file or directory, open '/var/task/chunks/schema.prisma'")

### DISCUSSION THREAD

**Discussion by bojidaryovchev** [prisma/prisma#10150] [Q&A]
Hello, so I followed this tutorial [AWS Amplify Next.js SSR](https://docs.amplify.aws/guides/hosting/nextjs/q/platform/js/) and deployed a Next.js app to AWS using Amplify. I added Prisma and it works fine locally, however I am getting this error in the deployed lambda function:

![prisma](https://user-images.githubusercontent.com/8954749/140654898-1f991fa4-3394-4b04-99e8-c70417bc1f04.png)

this is how the deployed lambda function looks like:

![image](https://user-images.githubusercontent.com/8954749/140654943-acfb1b65-10cc-4295-9a7b-14955e4c29ac.png)

apparently the ```LibraryEngine``` tries to look for ```schema.prisma``` in ```chunks``` folder but I have no idea how to copy it there.. so I am unable to figure out what should I do to make it work sadly..

I'd greatly appreciate it if you could help me out

**Comment by janpio** [comment]
Can you please open an issue with this information and supply all the information the bug issue template asks for? This should just work and no from from you should be necessary. If that is not the case, we have a bug that we should fix.

**Reply by bojidaryovchev** [reply]
@janpio https://github.com/prisma/prisma/issues/10200 there

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
